External/Notepad/Theme.swift

```diff
@@ -96,11 +96,11 @@ private extension Theme {
 private extension Theme {
 
     private static var fontSize: CGFloat {
```

Comment:
Perhaps this entire method could be taken away (and replaced by, directly, using fontSize, granted that we patch it up to return the default Size if nothing is set)